.github/workflows/release.yml

```
@@ -24,40 +24,47 @@ jobs:
             --target ${{ github.sha }}
 
 
-  build-jammy-package:
-    name: 'Build Ubuntu Jammy Package'
+  build-ubuntu-packages:
+    name: 'Build Ubuntu Packages'
```

For consistency with `test.yml`:

```suggestion
  build-ubuntu-package:
    name: 'Build Ubuntu package'
```

Consistency can also be ensured by doing the renaming in the other workflow. I prefer the singular form:
* `Build Ubuntu package (jammy, 15)`
* `Build Ubuntu package (noble, 17)`